dask/dataframe/backends.py

```diff
@@ -65,7 +66,16 @@ def make_meta_index(x, index=None):
     return x[0:0]
 
 
-@make_meta.register(object)
+meta_object_types = (pd.Series, pd.DataFrame, pd.Index, pd.MultiIndex)
+try:
+    import scipy.sparse as sp
+
+    meta_object_types += (sp.csr.csr_matrix,)
```

Comment:
Can use the top-level module for this

```suggestion
    meta_object_types += (sp.csr_matrix,)
```

Do we need other sparse matrices from SciPy? Also what about CuPy sparse matrices (cc @cjnolet)?